src/bun.js/bindings/sqlite/JSSQLStatement.cpp

```diff
@@ -2219,6 +2219,8 @@ JSC_DEFINE_HOST_FUNCTION(jsSQLStatementExecuteStatementFunctionRun, (JSC::JSGlob
         DO_REBIND(arg0);
     }
 
+    int total_changes_before = sqlite3_total_changes(castedThis->version_db->db);
```

Comment:
I'm not familiar with the sqlite api, was `sqlite3_step` or `sqlite3_stmt_readonly` resetting the total change count?

Reply:
If step only runs once then it would not report any changes